deployment/bin/backup-rds

```diff
@@ -0,0 +1,26 @@
+#!/bin/sh
+
+set -e
+
+# backup-data-volume /dev/xvdr
+
+RDS_ID=$(wget -q -O - http://instance-data/latest/user-data | grep 'RDS_ID=')
```

Comment:
I see. However, this hostname is not accessible from another ec2 instance:

ubuntu@ip-172-31-49-220:~$ curl http://instance-data/latest/user-data
curl: (7) Failed to connect to instance-data port 80: Connection timed out

I also couldn't find any documentation for it. Perhaps we should stick with the standard IP?